Src/PerformanceCollector/Shared/Implementation/StandardPerformanceCollector/StandardPerformanceCollector.cs

```diff
@@ -294,9 +294,9 @@ private void RegisterPerformanceCounter(string originalString, string reportAs,
                         usesInstanceNamePlaceholder,
```

Comment:
Can you pls confirm that we do not attempt to get the value too often. Use @tokaplan brain as he did those re-tries in case the counter will appear later